### PR TITLE
Chantal/card formatting

### LINKED ISSUE #13: User Story 1
As a user, when I visit the app, all movies should be displayed

### PR DESCRIPTION
**Pull request by GOECHA**
#Main Page Formatting

- [x] New feature (adds functionality)
- [x] # How Has This Been Tested?
​  - In the DOM
# How Has This Been Tested?
​
Please describe the tests that you ran to verify your changes.
​
- [x] I tested my changes in the browser
- [ ] I tested with Mocha/Chai
​ 
# Checklist:
​
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new bugs
- [] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

  - This formats the main page. Still need to update main page ehader image with random effect
  - Also, need to decide on format sizing for main page
  - I tried making a carousel. 
  - would like some insight, but after learning hooks it may he easier.

### CODE REVIEW

File: src/components/Carousel/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Card from '../Card/Card'
+import './Carousel.css'
+import logo from '../../images/rancid-tomatillos.png'
+
```
Comment (tfettig22) [inline src/components/Carousel/Carousel.js:5]: Is this like a carousel of movie cards that we will be able to scroll through? Love the idea! Nice job

**Review by tfettig22 — APPROVED** [review]
Nice job, thanks for getting so much done while I was gone! I think the carousel idea is great, that will look awesome for our page

File: src/components/App/App.js [hunk 2]
```diff
@@ -44,7 +45,16 @@ class App extends Component {
     return (
         <main className = 'App'>
           <Header />
-             { this.state.showModal.length ?(<Modal  props={this.state.showModal} backToHome={this.backToHome}/>) : (<MovieContainer movies={this.state.movies} selectAMovie={this.selectAMovie}/> )}   
+             { this.state.showModal.length ?(
```
Comment (nicktu12) [inline src/components/App/App.js:48]: nice job w/ conditional rendering here!

File: src/components/App/App.js [hunk 3]
```diff
@@ -44,7 +45,16 @@ class App extends Component {
...
+                <Modal  props={this.state.showModal} backToHome={this.backToHome}/> 
+                ) : (
+                  <>
```
Comment (nicktu12) [inline src/components/App/App.js:51]: What do react fragments compile down to? Would you want this to be a semantic HTML element? No wrong answers here just wondering!

File: src/components/Carousel/Carousel.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+const Carousel = ( {movies, selectAMovie} ) => {
+  
+  const movieCards = movies.map(movie => {
+    return (
+      <Card
+        logo={logo}
+        posterImage={movie.poster_path}
```
Comment (nicktu12) [inline src/components/Carousel/Carousel.js:13]: If you wanted to pass fewer props, you could send thru the whole `movie` object and break it apart in `Card` - up to you tho!

File: src/components/MovieContainer/MovieContainer.css [hunk 5]
```diff
@@ -1,9 +1,28 @@
+*{
+  border: solid 1px, purple;
```
Comment (nicktu12) [inline src/components/MovieContainer/MovieContainer.css:2]: What other CSS properties could be applied globally?

**Pull request merged**